docs/source/markdown/options/follow.md

```diff
@@ -6,6 +6,6 @@
 
 Follow log output.  Default is false.
 
-Note: If you are following a <<container|pod>> which is removed by `podman <<container|pod>> rm`
-or removed on exit (`podman run --rm ...`), then there is a chance that the log
+Note: Following a <<container|pod>> which is removed by `podman <<container|pod>> rm`
```

Comment:
(minor) I think '**WHEN** following ...' might be a little easier to understand.